install/install.sql

```diff
@@ -726,3 +735,26 @@ CREATE TABLE IF NOT EXISTS `v3_ipv4_mapping` (
     COLLATE = utf8mb4_unicode_ci;
 
 -- --------------------------------------------------------------------------------
+--
+-- Table structure for player ranking scores `v3_rankings`
+--
+CREATE TABLE IF NOT EXISTS `v3_rankings` (
+    `id`             INT UNSIGNED NOT NULL,
+    `username`       VARCHAR(30) CHARACTER SET ascii NOT NULL,
```

Comment:
This table should not have a `copy` of the `username` column from the `users` table. Instead use SQL JOINS (with the users table) when you want to get additional data about the user.

Also `id` should be renamed to `user_id`

Reply:
What is this supposed to mean?